packages/material-icon/index.scss

```diff
@@ -1 +1,9 @@
 @import url("https://fonts.googleapis.com/icon?family=Material+Icons");
+@import "@material/ripple/common";
+@import "@material/ripple/mixins";
+
+.material-icons {
+  @include mdc-ripple-surface;
```

Comment:
I don't think we should add mdc-ripple-surface to all material-icons. We should have another `material-icons--ripple-surface` class which this is limited to. And then update the MaterialIcon.js to apply this `material-icons--ripple-surface` as needed